tests/dummy/app/templates/application.hbs

```diff
@@ -31,13 +31,15 @@
       <div class="container-body">
         <div class="install-wrapper">
           <code class="install-cmd">ember install ember-cli-clipboard</code>
-          {{#copy-button
-            clipboardText="ember install ember-cli-clipboard"
-            success=(action "copyDirectSuccess")
-            error=(action "copyDirectError")
-          }}
-            {{fa-icon "copy" size=2}}
-          {{/copy-button}}
+          {{#if (is-clipboard-supported)}}
```

Comment:
Oh, I overlooked that. I reverted this change and added another example on 702ca8436c4ff1c22fd0f83285ae8bec01c96adf.